### PR TITLE
JAVA-18116 Review log statements for projects - Week 7 - 2023

### PR DESCRIPTION
**Pull request by jsgrah-spring**

### CODE REVIEW

File: patterns-modules/design-patterns-behavioral/src/test/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (anastasiosioannidis) [inline patterns-modules/design-patterns-behavioral/src/test/resources/logback.xml:1]: Please rename to logback-test.xml

File: persistence-modules/hibernate5/src/main/resources/log4j.properties [hunk 2]
```diff
@@ -1 +1,6 @@
 log4j.rootLogger=INFO, stdout
+
```
Comment (anastasiosioannidis) [inline persistence-modules/hibernate5/src/main/resources/log4j.properties:2]: We can revert these changes since hibernate5 module does not produce a lot of logs. Consequently, below changes may be out of scope

Comment (jsgrah-spring) [reply persistence-modules/hibernate5/src/main/resources/log4j.properties:2]: @anastasiosioannidis If we revert this change the test BootstrapAPIIntegrationTest will output the following  `log4j:ERROR Could not find value for key log4j.appender.stdout
log4j:ERROR Could not instantiate appender named "stdout".
log4j:WARN No appenders could be found for logger (org.hibernate.Version).
log4j:WARN Please initialize the log4j system properly.
`  I will go ahead and revert it, be i just need to mention the reason why it was added.

File: patterns-modules/design-patterns-behavioral/src/test/resources/logback.xml (hunk 1 above)
Comment (jsgrah-spring) [reply patterns-modules/design-patterns-behavioral/src/test/resources/logback.xml:1]: Renamed to logback-test.xml. I will update the PR at the end of the process. Thanks!

**Comment by jsgrah-spring** [comment]
Hi @anastasiosioannidis ,

I have update the PR with all changes that i found necessary. Please  have a look and let me know i f everything is OK.

Thanks
Joseph

File: core-java-modules/core-java-concurrency-advanced/src/main/java/com/baeldung/concurrent/countdownlatch/Worker.java [hunk 3]
```diff
@@ -15,7 +20,7 @@ public class Worker implements Runnable {
     @Override
     public void run() {
         // Do some work
-        System.out.println("Doing some logic");
```
Comment (anastasiosioannidis) [inline core-java-modules/core-java-concurrency-advanced/src/main/java/com/baeldung/concurrent/countdownlatch/Worker.java:18]: Is this change relevant to the task? As per description: "We need to ensure that our log statements generated **during the builds** ".
